.travis.yml

```diff
@@ -1,9 +1,19 @@
 language: python
-python:
-  - "2.7"
-install: "python setup.py install"
-script: "py.test tests/unit"
 branches:
     only:
         - master
         - dogfood
+services:
+  - postgres
+python:
+  - "2.7"
+env:
+  - DB=sqlite
+  - DB=postgres
+install:
+  - sh -c "if [ $DB = postgres ]; then pip install psycopg2; fi"
+  - python setup.py install
+before_script:
+  - sh -c "if [ $DB = postgres ]; then psql -c 'CREATE DATABASE haas_travis;' -U postgres; fi"
+  - sh -c "if [ $DB = postgres ]; then cp ci/testsuite.cfg.postgres testsuite.cfg; fi"
+script: "py.test tests/unit tests/stress.py"
```

Comment:
Could you split this into two commands, with `tests/unit` being run with with `-n auto`?

With the addition of `tests/stress.py` and the multiple DB types (postgres + sqlite), our CI builds are going from 1.5 mins to around 5. While I think running these are good changes, if we can take advantage of the [multiple cores](https://docs.travis-ci.com/user/migrating-from-legacy/) than folks can get responses a little faster.